R/tm_t_crosstable.R

```diff
@@ -169,7 +169,20 @@ srv_t_crosstable <- function(id, data, reporter, filter_panel_api, label, x, y,
   with_filter <- !missing(filter_panel_api) && inherits(filter_panel_api, "FilterPanelAPI")
   checkmate::assert_class(data, "tdata")
   moduleServer(id, function(input, output, session) {
-    selector_list <- teal.transform::data_extract_multiple_srv(data_extract = list(x = x, y = y), datasets = data)
+    selector_list <- teal.transform::data_extract_multiple_srv(
+      data_extract = list(x = x, y = y),
+      datasets = data,
+      select_validation_rule = list(
+        x = shinyvalidate::sv_required("Please define column for row variable that is not empty."),
```

Comment:
`sv_required` does not check whether a variable is "empty" (I assume this means having non-missing values)

Reply:
yes.